sdk/core/azure-core/tests/test_tracing_decorator.py

```diff
@@ -0,0 +1,34 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import unittest
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+from azure.core.tracing import common
+from azure.core.tracing import tracing_context, AbstractSpan
+from azure.core.settings import settings
+from opencensus.trace import tracer as tracer_module
+
+
+class TestCommon(unittest.TestCase):
+    def test_set_span_context(self):
+        wrapper = settings.tracing_implementation()
+        assert tracing_context.current_span.get() is None
+        assert wrapper.get_current_span() is None
+        parent = OpencenusWrapper()
+        common.set_span_contexts(parent)
+        assert parent.span_instance == wrapper.get_current_span()
+        assert tracing_context.current_span.get() == parent
+
+    def test_get_parent(self):
```

Comment:
Given the complexity of `get_parent`, I think it deserves more coverage.

Reply:
for sure - my tests have not been pushed up yet because I need the opencensus_impl to merge first. Checking that I make the correct children and stuff like that relies on opencensus.

Reply:
now you should see the more tests